config/300-azureservicebusqueuesource.yaml

```diff
@@ -52,19 +52,147 @@ spec:
             description: Desired state of the event source.
             type: object
             properties:
+              queueID:
```

Comment:
Might `queueID` be missing from the required fields?

Reply:
Nice catch